napari/_qt/widgets/qt_viewer_buttons.py

```diff
@@ -195,6 +210,133 @@ def open_perspective_popup(self):
         pop.frame.setLayout(layout)
         pop.show_above_mouse()
 
+    def _open_grid_popup(self):
+        """Open grid options pop up widget."""
+
+        # widgets
+        popup = QtPopup(self)
+        grid_stride = QtSpinBox(popup)
+        grid_width = QtSpinBox(popup)
+        grid_height = QtSpinBox(popup)
+        shape_help_symbol = QLabel(self)
+        stride_help_symbol = QLabel(self)
+        blank = QLabel(self)  # helps with placing help symbols.
+
+        shape_help_msg = trans._(
+            'Number of rows and columns in the grid. A value of -1 for either or '
```

Comment:
@goanpeca, I thought we couldn't break `trans_` strings?

Reply:
We cannot but I usually fix them prior to a release 🤷🏻‍♂️